### ISSUE TITLE
Add excluded-properties to no-vendor-prefixes rule

### ISSUE THREAD

**Issue by keeganstreet** [sasstools/sass-lint#705]
The `no-vendor-prefixes` rule allows us to exclude certain identifiers, such as `webkit` or `moz`. I would like to be able to configure the rule to exclude specific properties without excluding the entire identifier.

Example:
````yml
no-vendor-prefix:
  - 1
  -
    'excluded-properties':
      - '-webkit-tap-highlight-color'
````

The following would be allowed:

````scss
html {
	-webkit-tap-highlight-color: $link-color-hover;
}
````

The following would be disallowed, because the `excluded-identifiers` option has not been used:

````scss
.round {
	-webkit-border-radius: 12px;
}
````

Do you think this would be a useful option?

**Comment by egaudette** [comment]
Duplicate of #702 

Your suggested code addition is exactly what I was hoping would be included in the linting.

**Comment by DanPurdy** [comment]
I've updated #702 with a comment/suggestion if you'd like to choose 👍 will close this as a duplicate
